Common/Data/Custom/Robintrack/RobintrackHoldings.cs

```diff
@@ -0,0 +1,164 @@
+        /// </summary>
+        public decimal UniverseHoldingPercent => UsersHolding / TotalUniqueHoldings;
+
+        /// <summary>
+        /// Alias for <see cref="UsersHolding"/>
+        /// </summary>
+        public override decimal Value
+        {
+            get { return UsersHolding; }
+            set { UsersHolding = (int)value; }
+        }
+
+        /// <summary>
+        /// Gets the source of the file
+        /// </summary>
+        /// <param name="config">Subscription config</param>
+        /// <param name="date">Data for the given date to read</param>
+        /// <param name="isLiveMode">Is live mode</param>
+        /// <returns>Subscription data source</returns>
+        public override SubscriptionDataSource GetSource(SubscriptionDataConfig config, DateTime date, bool isLiveMode)
+        {
+            return new SubscriptionDataSource(
+                Path.Combine(
+                    Globals.DataFolder,
+                    "alternative",
+                    "robintrack",
+                    $"{config.Symbol.Underlying.Value.ToLowerInvariant()}.csv"
+                ),
+                SubscriptionTransportMedium.LocalFile,
+                FileFormat.Csv
+            );
+        }
+
+        /// <summary>
+        /// Reads the data from the source provided in <see cref="GetSource(SubscriptionDataConfig, DateTime, bool)"/>
+        /// and converts it into an instance of this class (<see cref="RobintrackHoldings"/>).
+        /// </summary>
+        /// <param name="config">Subscription config</param>
+        /// <param name="line">Line to parse</param>
+        /// <param name="date">Date that the data is being read</param>
+        /// <param name="isLiveMode">Is live mode</param>
+        /// <returns>Instance of <see cref="RobintrackHoldings"/> casted as <see cref="BaseData"/></returns>
+        public override BaseData Reader(SubscriptionDataConfig config, string line, DateTime date, bool isLiveMode)
+        {
+            var instance = Read(line);
+            instance.Symbol = config.Symbol;
+
+            return instance;
+        }
+
+        /// <summary>
+        /// Creates a clone of the current object
+        /// </summary>
+        /// <returns>Copy of the current object</returns>
+        public override BaseData Clone()
+        {
+            return new RobintrackHoldings
+            {
+                EndTime = EndTime,
+                Symbol = Symbol,
+                Value = Value,
+                TotalUniqueHoldings = TotalUniqueHoldings,
+            };
+        }
+
+        /// <summary>
+        /// Parses a line of Robintrack data with an optional dateFormat
+        /// </summary>
+        /// <param name="line">Line to parse</param>
+        /// <param name="dateFormat">Date format to parse timestamp in</param>
+        /// <returns>Instance of this class</returns>
+        public static RobintrackHoldings Read(string line, string dateFormat = "yyyyMMdd HH:mm:ss")
```

Comment:
Would be nice to see some simple read unit test and maybe a json round trip too?